src/pages/index.jsx

```diff
@@ -87,6 +88,19 @@ const SupportItems = [
   },
 ];
 
+const prices = {
+  annual: {
+    starter: 0,
+    premium: 499,
+    enterprise: 1999,
+  },
+  monthly: {
+    starter: 0,
+    premium: 750,
```

Comment:
Technically 749 (that's what Trivalence pays for monthly). https://phasetwo.io/docs/service-agreement/#fees